spec/bitcoin.did

```diff
@@ -0,0 +1,69 @@
+type Satoshi = nat64;
+
+type OutPoint = record {
+  txid : blob;
+  vout : nat32
+};
+
+type Utxo = record {
+  outpoint: OutPoint;
+  value: Satoshi;
+  height: nat32;
+};
+
+type Pagination = record {
+  tip_hash : BlockHash;
+  offset: opt nat32;
+  limit: opt nat32;
+}
+
+type GetUtxosRequest = record {
+  address : text;
+  filter: opt variant {
+    MinConfirmations: nat32;
+    Pagination: Pagination;
+  };
+};
+
+type GetUtxosError = variant {
+  MalformedAddress;
+  // More error types to be added here.
+};
+
+type GetBalanceRequest = record {
+  address : text;
+  min_confirmations: opt nat32;
+};
+
+type GetBalanceError = variant {
+  MalformedAddress;
+  // More error types to be added here.
+};
+
+type SendTransactionError = variant {
+  MalformedTransaction;
+  // More error types to be added here.
```

Comment:
Should we remove these comments from the candid? We've already established that these can be extended in a backward-compatible way.